src/Features/CSharp/Portable/Completion/CSharpCompletionService.cs

```diff
@@ -25,8 +25,16 @@ public ILanguageService CreateLanguageService(HostLanguageServices languageServi
 
     internal class CSharpCompletionService : CommonCompletionService
     {
-        private readonly ImmutableArray<CompletionProvider> _defaultCompletionProviders =
-            ImmutableArray.Create<CompletionProvider>(
+        private readonly ImmutableArray<CompletionProvider> _defaultCompletionProviders;
+
+        private readonly Workspace _workspace;
```

Comment:
still not sure why we need a workspace.

Reply:
I didn't change this, the field was there before. I just moved it around.